modules/ocl-loader/src/main/java/org/eclipse/xpanse/modules/ocl/loader/data/models/vo/ProviderOclVo.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.ocl.loader.data.models.vo;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import lombok.Data;
+import org.eclipse.xpanse.modules.ocl.loader.data.models.enums.Csp;
+
+
+/**
+ * Define view object for UI Client query registered service by csp.
+ */
+@Data
+public class ProviderOclVo {
+
+    @Schema(description = "The Cloud Service Provider. valid values: aws, azure, alibaba, huawei")
```

Comment:
we can avoid providing enum values in description again. There are few other places also where we are doing it. we can remove it everywhere.

Reply:
OK. I removed the enum values from description.